backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5577960_sys_gh10568_AddOrgFilter.sql

```diff
@@ -0,0 +1,42 @@
+-- 2021-02-01T09:39:40.879Z
+-- URL zum Konzept
+INSERT INTO AD_Process_Para (AD_Client_ID,AD_Element_ID,AD_Org_ID,AD_Process_ID,AD_Process_Para_ID,AD_Reference_ID,ColumnName,Created,CreatedBy,DefaultValue,Description,EntityType,FieldLength,Help,IsActive,IsAutocomplete,IsCentrallyMaintained,IsEncrypted,IsMandatory,IsRange,Name,SeqNo,Updated,UpdatedBy) VALUES (0,113,0,584656,541919,19,'AD_Org_ID',TO_TIMESTAMP('2021-02-01 10:39:35','YYYY-MM-DD HH24:MI:SS'),100,'1000000','Organisatorische Einheit des Mandanten','U',0,'Eine Organisation ist ein Bereich ihres Mandanten - z.B. Laden oder Abteilung. Sie kĂ¶nnen Daten ĂĽber Organisationen hinweg gemeinsam verwenden.','Y','N','Y','N','Y','N','Sektion',5,TO_TIMESTAMP('2021-02-01 10:39:35','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
are the new parameters mandatory? if not, i think they should be..

Reply:
yes, they are